packages/dds/shared-object-base/api-report/shared-object-base.legacy.alpha.api.md

```diff
@@ -62,7 +62,7 @@ export abstract class SharedObjectCore<TEvent extends ISharedObjectEvents = ISha
     protected get deltaManager(): IDeltaManager<ISequencedDocumentMessage, IDocumentMessage>;
     protected didAttach(): void;
     protected dirty(): void;
-    emit(event: EventEmitterEventType, ...args: any[]): boolean;
+    emit(event: EventEmitterEventType, ...args: unknown[]): boolean;
```

Comment:
This pattern is actually problematic and will cause some cases would have previously compiled to no longer compile. The eslint config _should_ be okay with `any[]` in this case. Did it complain about this?